iceoryx_examples/icedelivery/iox_publisher_untyped_modern.cpp

```diff
@@ -29,25 +29,40 @@ static void sigHandler(int f_sig [[gnu::unused]])
     killswitch = true;
 }
 
-int main(int argc, char* argv[])
+int main()
 {
     // Register sigHandler for SIGINT
     signal(SIGINT, sigHandler);
 
-    iox::runtime::PoshRuntime::getInstance("/iox-ex-publisher-modern");
+    iox::runtime::PoshRuntime::getInstance("/iox-ex-publisher-untyped-modern");
 
     auto untypedPublisher = iox::popo::UntypedPublisher({"Odometry", "Position", "Vehicle"});
     untypedPublisher.offer();
 
     float_t ct = 0.0;
     while (!killswitch)
     {
+        ++ct;
+
+        // API Usage #1
+        //  * Loaned sample can be held until ready to publish
+        auto maybeSample = untypedPublisher.loan(128);
```

Comment:
in the typed publisher this was called `result` instead of `maybeSample`. Should it be harmonized?

Reply:
Optional vs. expected.

Reply:
Why the difference? I guess this loan can also fail

Reply:
Ah I got confused (looked at the wrong line in my IDE). this call is in fact an expected and should be named result.